std_msgs/String object_class
int16 object_index
geometry_msgs/Point pos
float32 radius
geometry_msgs/Point orientation